cassandra/tablets.py

```diff
@@ -51,7 +58,42 @@ def get_tablet_for_key(self, keyspace, table, t):
             return tablet[id]
         return None
 
+    def drop_tablet(self, keyspace: str, table: str = None):
+        print("drop tablets for ", keyspace, table)
+        with self._lock:
+            if table is not None:
+                self._tablets.pop((keyspace, table), None)
+                return
+
+            to_be_deleted = []
+            for key in self._tablets.keys():
+                if key[0] == keyspace:
+                    to_be_deleted.append(key)
+
+            for key in to_be_deleted:
+                del self._tablets[key]
+
+    def drop_tablet_by_host_id(self, host_id: UUID):
+        print("drop tablets for ", host_id)
+        if host_id is None:
+            return
```

Comment:
🔧 Either change the type hint for `host_id` or drop the `None` check - why check for `None` if type of `host_id` is `UUID` (and not `UUID | None`)?

Reply:
`|` is not supported by python 3.8